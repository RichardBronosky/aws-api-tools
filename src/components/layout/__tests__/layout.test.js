import React from 'react'
import renderer from 'react-test-renderer'

jest.mock('../../shared/instant-search-router', () => 'InstantSearchRouter')
jest.mock('../layout-children', () => 'LayoutChildren')

import Layout from '../layout'

describe('Layout', () => {
  it('renders correctly with default props', () => {
    const tree = renderer.create(<Layout />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
