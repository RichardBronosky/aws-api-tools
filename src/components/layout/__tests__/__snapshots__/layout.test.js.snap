// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly with default props 1`] = `
<div>
  <a
    className="skip"
    href="#content"
  >
    Skip to Main Content
  </a>
  <div
    className="browser-detect"
  />
  <InstantSearchRouter>
    <LayoutChildren />
  </InstantSearchRouter>
</div>
`;
